.travis.yml

```diff
@@ -21,6 +21,7 @@ matrix:
     os: windows
 
 script:
+  - cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose
```

Comment:
Maybe try this? not sure if it's the exact syntax that travis expects so you'll have to check the output on travis
`if [ ${TRAVIS_RUST_VERSION} == "1.22.0" ]; then cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose; fi`

Reply:
Seems to work and no worries :)